node/runtime/pangoro/src/lib.rs

```diff
@@ -693,11 +693,23 @@ sp_api::impl_runtime_apis! {
 			use frame_system_benchmarking::Pallet as SystemBench;
 			use baseline::Pallet as BaselineBench;
 
-
 			impl frame_system_benchmarking::Config for Runtime {}
 			impl baseline::Config for Runtime {}
 
-			let whitelist: Vec<TrackedStorageKey> = vec![];
+			let whitelist: Vec<TrackedStorageKey> = vec![
```

Comment:
What does the `whitelist` mean? Or any docs about this?

Reply:
They have a cache.
There is no need to count r/w for the global-dup storage.